polyinterface/__init__.py

```diff
@@ -1,6 +1,6 @@
 from .polyinterface import Interface, Node, Controller, LOGGER, unload_interface, get_network_interface
 
-__version__ = '2.0.32'
+__version__ = '2.0.33'
```

Comment:
Thought 34 is the current one?
```suggestion
__version__ = '2.0.35'
```

Reply:
Fixed, thank you!